CS3219-AY2425S1/cs3219-ay2425s1-project-g25
# Implement user authentication with jwt access and refresh tokens

* Access token: Stored in localstorage, send as bearer token for protected routes by an axios interceptor
* Refresh token: Stored in httpOnly cookies
* Access token expiry and refresh is taken care by axios interceptors

To test:
* Refer to .env.example (generate random jwt secret)
* Inspect network: Every `/verify-token` that returns 401 should match with a successful `/refresh`
